File: scripts/read_template.js
const { Pool } = require('pg');
const fs = require('fs');

const pool = new Pool({
    host: 'localhost',
    port: 15432,
    database: 'net_imobiliaria',
    user: 'postgres',
    password: 'postgres',
});

async function readTemplate() {
    try {
        const res = await pool.query("SELECT html_content, variables FROM email_templates WHERE name = 'password_reset'");
        if (res.rows.length > 0) {
            const content = res.rows[0].html_content;
            fs.writeFileSync('scripts/template_content.html', content);
            console.log("Written to scripts/template_content.html");
        } else {
            console.log("Template not found");
        }
    } catch (err) {
        console.error(err);
    } finally {
        await pool.end();
    }
}

readTemplate();

File: scripts/read_template.py
import psycopg2
import os

try:
    conn = psycopg2.connect(
        host="localhost",
        port="15432",
        database="net_imobiliaria",
        user="postgres",
        password="postgres"
    )
    cur = conn.cursor()
    cur.execute("SELECT html_content, variables FROM email_templates WHERE name = 'password_reset'")
    row = cur.fetchone()
    if row:
        print("--- HTML CONTENT ---")
        print(row[0])
        print("--- VARIABLES ---")
        print(row[1])
    else:
        print("Template not found")
    cur.close()
    conn.close()
except Exception as e:
    print(f"Error: {e}")
